container/container.go

```diff
@@ -13,11 +13,33 @@ import (
 // Our own narrowly-scoped interface for Docker client
 type DockerClient interface {
 	PullImage(docker.PullImageOptions, docker.AuthConfiguration) error
+	ListImages(docker.ListImagesOptions) ([]docker.APIImages, error)
+}
+
+// Loop through all the images and see if we have one with a match
+// on this repo image:tag combination.
+func CheckImage(client DockerClient, taskInfo *mesos.TaskInfo) bool {
+	images, err := client.ListImages(docker.ListImagesOptions{All: false})
```

Comment:
Format the line?  If you mean code formatting, then no, it's formatted with the go tools.